### ISSUE TITLE
Model Position/Orientation/Scale Manipulation

### ISSUE THREAD

**Issue by IanNBackus** [Extended-reality-Lab/Immersive-Animal-Anatomy#2]
User Story: As a student learning anatomy, I want to be able to change the position, scale, and orientation of the model so that I can most easily observe its various parts.

Background: 
-We want this feature due to the fact that the model needs to be able to be viewable from a functionally infinite amount of possible angles, as a user must be able to easily observe any obscure portion of the anatomy clearly.
-This feature is intended for users attempting to learn and better understand the anatomy of these animals by interacting with them in a hands-on setting with strong visuals.
-The main data we will need for a feature like this is the data in the form of the model being manipulated, and the data from the input devices used to manipulate the model (in this case the controllers), and we indeed have access to both of these data points.

Requirements:
   **Functional:**
-User must be able to alter the position of the model, orientation, and scale of the model with their Oculus Quest 2 controllers
-User must be able to perform these manipulations at any time when interacting with the model.
   **Non-Functional:**
-Manipulation of the model must be intuitive and easy to perform without mis-input.

Design resources:
-The manipulation should somewhat mimic common gestures used when interacting with similar systems, such as changing the scale of a photo on a phone, or changing the orientation of something in a 3D viewer.

Dependencies:
   **Blocked By:**
-Importing of a chosen model into the Unity Project
-Movement system for navigating the VR space being finished
   **Blocking:**
-This feature does not specifically block any other feature in terms of the feature being possible to use (as in VR a user can move around the model as need be), but other features such as highlighting, annotating, viewing, etc. will be made much easier with the completion of this user story.

Subtasks:
-Decide on user-friendly control scheme for 3-way model model manipulation
-Create system for changing model scale
-Create system for changing model orientation
-Create system for changing model position

Estimate: 2 weeks

Open Questions: What _exactly_ with the gestures to complete this be?

Acceptance Criteria:
-Users can alter the orientation of the model
-Users can alter the position of the model
-Users can alter the scale of the model

DoD: 
-Code-reviewed and merged into the main branch
-Feature approved by the project partner
-Demo video created
-Feature has been tested to be functional and performant on the headset
